docs/user-manual.rst

```diff
@@ -915,6 +915,108 @@ will compute a boolean that tells you whether the field "000000" passes
+kind of functions are called **t-norms** and **t-conorms**.
+
+You can find more information about t-norms and t-conorms in the
+following links:
+
+- `Wikipedia: t-norms and t-conorms <https://en.wikipedia.org/wiki/T-norm>`__
+- `Wikipedia: Construction of t-norms <https://en.wikipedia.org/wiki/Construction_of_t-norms>`__
+
+All the norms need, at least, two arguments. The 2 fields where
+t-norms will be applied to. You can specify them with an string
+representing their field id or its field name. However, you can also
+use this norms with two real numbers as arguments, and the norms will
+be applied to these numbers:
+
+::
+
+        (tnorm-min "mycolumnname1" "mycolumnname2")
+        (tnorm-min "000002" "000001")
+        (tnorm-min 0.70 0.24)
+
+
+Fields used as operands must contain real values between 0 and 1. As
+they are logical values it doesn't make sense having values outside
+this range. If you pass a field to the norms with more than 80% of
+its values outside this range, an exception will be raised.
+When some sparse out-of-range values are found during calculations,
+the generated field will contain a missing value for this specific
+row.
+
+Consider normalizing or truncating your fields, before passing
+them to the fuzzy logic norms:
+
+.. code:: lisp
+
+       (max 0 (min 1 (field "000001"))) ;; Truncating field
+       (normalize "000001") ;; Normalizing field
+
+You could do something like this:
+
+.. code:: lisp
+
+       (tnorm-min (normalize "000001")  (normalize "000002"))
+
+
+Basic T-norms
+~~~~~~~~~~~~~
+
+We provide the following basic t-norms.  All of them need 2
+parameters.
+
+::
+
+        (tnorm-min <f1> <f2>) ;; Minimum t-norm. Also called the Gödel t-norm.
+        (tnorm-product <f1> <f2>) ;; Product t-norm. The ordinary product of real numbers.
+        (tnorm-lukasiewicz <f1> <f2>) ;; Łukasiewicz t-norm.
+        (tnorm-drastic <f1> <f2>) ;; Drastic t-norm
+        (tnorm-nilpotent-min <f1> <f2>) ;; Nilpotent minimum t-norm
+
+
+
+Basic T-conorms
+~~~~~~~~~~~~~~~
+
+We provide the following basic t-conorms.  All of them need 2
+parameters.
+
+::
+
+        (tconorm-max <f1> <f2>) ;; Maximum t-norm. Dual to the minimum t-norm, is the smallest t-conorm.
+         (tconorm-probabilistic <f1> <f2>) ;; Probabilistic t-norm. It's dual to the product t-norm.
+         (tconorm-bounded <f1> <f2>) ;; Bounded t-norm. It'ss dual to the Łukasiewicz t-norm.
+         (tconorm-drastic <f1> <f2>) ;; Drastic t-conorm. It's dual to the drastic t-norm.
+         (tconorm-nilpotent-max <f1> <f2>) ;; Nilpotent maximum t-conorm. It's dual to the nilpotent minumum.
+         (tconorm-einstein-sum <f1> <f2>) ;; Einstein t-conorm. It's a dual to one of the Hamacher t-norms.
+
+
+Parametric T-norms
+~~~~~~~~~~~~~~~~~~
+
+We provide the following parametrized t-norms.  All of them need 3
+parameters, the two fields were t-norms will be applied, and the
+parameter ``p`` that changes their behavior.. This parameter must be a
```

Comment:
can we say what "p" is? (at least give it a name), and be a bit more
explicit about what "changes their behaviour" means (qualitatively, at least)